### DISCUSSION TITLE
Can I enable any verbose or debug output for the extension?

### DISCUSSION THREAD

**Discussion by holvi-sebastian** [vitest-dev/vscode#453] [Q&A]
I am trying to debug a problem where the extension's continuous mode doesn't actually run any tests on changes.

Vitest in terminal runs fine, and watches fine. If I tell the extension to run all tests, they all run fine. In the extension's output, it seems to be detecting changes to files correctly.

It just never runs tests in response to changes in continuous mode.

I don't see any errors in the extension's output.

Is there any way to configure a debug or trace mode for the extension, so I can try to figure out why no tests are being run on change?

**Comment by sheremet-va** [comment]
Yes, you can add `VITEST_VSCODE_DEBUG: 'true'` to `vitest.nodeEnv` in your `settings.json`:

```json
{
  "vitest.nodeEnv": {
    "VITEST_VSCODE_DEBUG": "true",
  }
}
```

**Reply by sheremet-va** [reply]
In 1.2.1 there is a `vitest.logLevel` option (when it's released)
